### PR TITLE
[stable/postgreslq] Improve service configuration

### PR DESCRIPTION
**Pull request by juan131**
#### What this PR does / why we need it:

This PR improves the service configuration by providing new parameters that can be used to customize the K8s service to deploy.

### CODE REVIEW

File: stable/postgresql/README.md [hunk 1]
```diff
@@ -45,59 +45,63 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `tolerations`                        | Toleration labels for pod assignment               | `[]`                                                      |
+| `terminationGracePeriodSeconds`      | Seconds the pod needs to terminate gracefully      | `nil`                                                     |
+| `resources`                          | CPU/Memory resource requests/limits                | Memory: `256Mi`, CPU: `250m`                              |
+| `securityContext.enabled`            | Enable security context                            | `true`                                                    |
+| `securityContext.fsGroup`            | Group ID for the container                         | `1001`                                                    |
+| `securityContext.runAsUser`          | User ID for the container                          | `1001`                                                    |
+| `livenessProbe.enabled`              | would you like a livessProbed to be enabled        |  `true`                                                   |
```
Comment (carrodher) [inline stable/postgresql/README.md]:
```suggestion
| `livenessProbe.enabled`              | Would you like a livenessProbe to be enabled        |  `true`                                                   |
```

File: stable/postgresql/README.md [hunk 2]
```diff
@@ -45,59 +45,63 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `networkPolicy.allowExternal`        | Don't require client label for connections         | `true`                                                    |
+| `livenessProbe.initialDelaySeconds`  | Delay before liveness probe is initiated           |  30                                                       |
+| `livenessProbe.periodSeconds`        | How often to perform the probe                     |  10                                                       |
+| `livenessProbe.timeoutSeconds`       | When the probe times out                           |  5                                                        |
+| `livenessProbe.failureThreshold`     | Minimum consecutive failures for the probe to be considered failed after having succeeded.    |  6             |
+| `livenessProbe.successThreshold`     | Minimum consecutive successes for the probe to be considered successful after having failed   |  1             |
+| `readinessProbe.enabled`             | would you like a readinessProbe to be enabled      |  `true`                                                   |
```
Comment (carrodher) [inline stable/postgresql/README.md:89]:
```suggestion
| `readinessProbe.enabled`             | Would you like a readinessProbe to be enabled      |  `true`                                                   |
```

File: stable/postgresql/README.md [hunk 3]
```diff
@@ -45,59 +45,63 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `readinessProbe.successThreshold`    | Minimum consecutive successes for the probe to be considered successful after having failed   |  1             |
+| `metrics.enabled`                    | Start a prometheus exporter                        | `false`                                                   |
+| `metrics.service.type`               | Kubernetes Service type                            |  `ClusterIP`                                              |
+| `metrics.service.annotatios`         | Additional annotations for metrics exporter pod    |  `{}`                                                     |
+| `metrics.service.loadBalancerIP`     | loadBalancerIP if redis metrics service type is `LoadBalancer` | `nil`                                         |
+| `metrics.image.registry`             | PostgreSQL image registry                          | `docker.io`                                               |
+| `metrics.image.repository`           | PostgreSQL Image name                              | `wrouesnel/postgres_exporter`                             |
```
Comment (carrodher) [inline stable/postgresql/README.md:100]:
```suggestion
| `metrics.image.repository`           | PostgreSQL image name                              | `wrouesnel/postgres_exporter`                             |
```

File: stable/postgresql/README.md [hunk 4]
```diff
@@ -45,59 +45,63 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `metrics.image.tag`                  | PostgreSQL Image tag                               | `{VERSION}`                                               |
```
Comment (carrodher) [inline stable/postgresql/README.md:101]:
```suggestion
| `metrics.image.tag`                  | PostgreSQL image tag                               | `{VERSION}`                                               |
```

File: stable/postgresql/README.md [hunk 5]
```diff
@@ -45,59 +45,63 @@ The command removes all the Kubernetes components associated with the chart and
...
-| `metrics.image.pullSecrets`           | Specify image pull secrets                                                                    | `nil` (does not add image pull secrets to deployed pods)  |
-| `extraEnv`                            | Any extra environment variables you would like to pass on to the pod                          | {}                                                        |
+|         Parameter                    |                Description                         |                            Default                        |
+|--------------------------------------|----------------------------------------------------|---------------------------------------------------------- |
+| `global.imageRegistry`               | Global Docker image registry                       | `nil`                                                     |
+| `image.registry`                     | PostgreSQL image registry                          | `docker.io`                                               |
+| `image.repository`                   | PostgreSQL Image name                              | `bitnami/postgresql`                                      |
```
Comment (carrodher) [inline stable/postgresql/README.md:52]:
```suggestion
| `image.repository`                   | PostgreSQL image name                              | `bitnami/postgresql`                                      |
```

File: stable/postgresql/README.md [hunk 6]
```diff
@@ -45,59 +45,63 @@ The command removes all the Kubernetes components associated with the chart and
...
+| `image.tag`                          | PostgreSQL Image tag                               | `{VERSION}`                                               |
```
Comment (carrodher) [inline stable/postgresql/README.md:53]:
```suggestion
| `image.tag`                          | PostgreSQL image tag                               | `{VERSION}`                                               |
```

**Review by carrodher — COMMENTED** [review]
Small suggestions related to the README

File: stable/postgresql/README.md (hunk 2 above)
Comment (juan131) [reply stable/postgresql/README.md:89]: 👍

File: stable/postgresql/README.md (hunk 3 above)
Comment (juan131) [reply stable/postgresql/README.md:100]: 👍

File: stable/postgresql/README.md (hunk 4 above)
Comment (juan131) [reply stable/postgresql/README.md:101]: 👍

File: stable/postgresql/README.md (hunk 5 above)
Comment (juan131) [reply stable/postgresql/README.md:52]: 👍

File: stable/postgresql/README.md (hunk 6 above)
Comment (juan131) [reply stable/postgresql/README.md:53]: 👍

**Pull request merged**